units/host_allocator.go

```diff
@@ -345,6 +351,63 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	)
 }
 
+func (j *hostAllocatorJob) handleSingleTaskDistroAllocation(ctx context.Context, distro *distro.Distro, distroQueueInfo model.DistroQueueInfo, existingHosts host.HostGroup) {
```

Comment:
rather than have this as its own separate function with some duplicated logic and logs, I'd prefer to skip the host allocator function in the job if the distro is a single task distro.

Reply:
updated